google-cloud-firestore/samples/CONTRIBUTING.md

```diff
@@ -0,0 +1,289 @@
+directory:
+
+```sh
+$ bundle exec rake rubocop
+```
+
+In the rare case that you need to override the existing Rubocop configuration for the samples in order to accommodate
+your changes, you can do so by updating [.rubocop.yml](.rubocop.yml) in the samples directory.
+
+### Acceptance Tests
+
+The acceptance tests (a.k.a. integration tests) ensure that the samples work correctly against the live service API.
+To configure your Google Cloud project, see [Acceptance tests setup](#acceptance-tests-setup) above.
+
+**Warning: You may incur charges while running the acceptance tests against your Google Cloud project.**
+
+The acceptance tests are based on the [minitest](https://github.com/seattlerb/minitest) library, including
+[specs](https://github.com/seattlerb/minitest#specs-) and [minitest-focus](https://github.com/seattlerb/minitest-focus).
+Mocks are not generally used in acceptance tests. Because the acceptance test suite is often time-consuming to run in
+its entirety, during development or debugging you may want to isolate one or more of the tests by placing the `focus`
+keyword just above the test declaration. (See [minitest-focus](https://github.com/seattlerb/minitest-focus) for
+details.)
+
+To run the acceptance tests, run the following command in the `google-cloud-firestore` directory:
+
+``` sh
+$ bundle exec rake samples
+```
+
+Even though you invoke the `samples` rake task in the `google-cloud-firestore` directory rather than the
+`google-cloud-firestore/samples` directory, the task creates a new Bundler environment that isolates the
+`google-cloud-firestore/samples` directory from its parent. The task then runs `bundle update` in the
+`google-cloud-firestore/samples` directory before running the tests. By default, this environment loads the latest
+*published* version of the `google-cloud-firestore` gem. If your samples depend on changes to source code in your local
+`google-cloud-firestore` directory, run this command in the `google-cloud-firestore` directory:
+
+``` sh
+$ bundle exec rake samples:master
+```
+
+There may be tests that usually pass but fail occasionally due to issues like eventual consistency. However, please
+ensure that you do successfully run acceptance tests for any samples covered by your pull request.
+
+## Make changes
+
+All contributions should include new or updated tests to ensure that the contributed code behaves as expected.
+
+To run a single test, temporarily add the `focus` keyword just above the test declaration. (See
+[minitest-focus](https://github.com/seattlerb/minitest-focus) for details.) When you are done developing, be sure to
+remove any usages of the `focus` keyword from your tests.
+
+## Commit changes
+
+Commit your changes using [conventional commits](https://www.conventionalcommits.org/), making sure to include the
+associated GitHub issue number. Below is an example of the `chore` type commit that should be used with samples changes
```

Comment:
Just state outright something like "Please use `chore:` as the commit type for all samples changes, to prevent them from triggering a library release."

Reply:
Updated, thanks!